models/Table.js

```diff
@@ -0,0 +1,22 @@
+import mongoose from "mongoose";
+const { Schema } = mongoose;
+
+const tableSchema = new mongoose.Schema({
+  id: Schema.Types.ObjectId,
+  name: {
```

Comment:
add restaurant id here

Reply:
Instead of the id that is now, should I add the id of the restaurant?